### PR TITLE
Update Installation.md for Mobile users [Fixes #1123]

### LINKED ISSUE #1123: No proper installation instruction in Talawa API Installation Docs for Mobile Users
**Is your feature request related to a problem? Please describe.**
There are no proper instructions for using the Local Talawa API Instance inside the Talawa Mobile App developers.

For example, we have Talawa Api running inside local machine and we can access it using ```localhost:{PORT_NUMBER}/graphql```

Now, how can we access this URL inside the Mobile App running on a real device or, inside the Android Virtual Device. Like we can use ```10.0.2.2``` for AVD, and IPV4 address for a real device.

**Describe the solution you'd like**
Changes to installation docs need to be done for users to figure out this thing.

**Describe alternatives you've considered**
Alternatively, developers can search the web for this thing, but, it is not what should be done.

**Approach to be followed (optional)**
A clear and concise description of approach to be followed.

**Additional context**
No

### PR DESCRIPTION
**Pull request by anshgoyalevil**
**What kind of change does this PR introduce?**
This PR introduces some additions to the installation.md file for setting up talawa-api for mobile developers.

**Issue Number:**

Fixes #1123

**Did you add tests for your changes?**
Not required

**Snapshots/Videos:**
Not required

**If relevant, did you update the documentation?**
Yes

**Summary**
This PR introduces some additions to the installation.md file for setting up talawa-api for mobile developers.

**Does this PR introduce a breaking change?**
No

**Other information**
No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -338,9 +338,27 @@ GraphQL endpoint for handling `subscriptions` is this:
...
+- If Talawa Mobile App is installed on Android Virtual Device (AVD): Use the following URL:
+```
+    http://10.0.2.2:4000/graphql
+```
+- If Talawa Mobile App is installed on a Read Android Device: Use the following method to get URL.
+    - Open Command Prompt in Windows, or Terminal in Linux/OSX
+    - Enter ```ipconfig``` (For Windows Users) or ```ifconfig``` (For Linux/OSX Users)
+    - Your Mobile and Computer (On which API server is running) must be on same Wifi Network. Use Mobile Hotspot to connect your computer to internet in case you don't have access to a Wifi Router.
+    - Search for the ```Wireless LAN adapter Wi-Fi:``` and then copy ```IPv4 Address```, like in image below:
+    - ![image](https://user-images.githubusercontent.com/94157520/223067277-d1e40093-a018-479c-bf3c-07749fbb75b9.png)
```
Comment (palisadoes) [inline INSTALLATION.md:352]: User an image on the repo. We want to make sure that this works even when the person is working completely offline.

File: INSTALLATION.md [hunk 2]
```diff
@@ -338,9 +338,27 @@ GraphQL endpoint for handling `subscriptions` is this:
...
 
-The Organization URL for Talawa mobile app developers to use is:
+The Organization URL for Talawa mobile app developers will depend upon the device on which Mobile app is installed.
 
-       http://localhost:4000/graphql/
+- If Talawa Mobile App is installed on Android Virtual Device (AVD): Use the following URL:
+```
+    http://10.0.2.2:4000/graphql
+```
+- If Talawa Mobile App is installed on a Read Android Device: Use the following method to get URL.
```
Comment (palisadoes) [inline INSTALLATION.md:347]: Typographical error.  Needs fixing

File: INSTALLATION.md [hunk 3]
```diff
@@ -338,9 +338,27 @@ GraphQL endpoint for handling `subscriptions` is this:
 
 ## For Mobile App Developers
 
-The Organization URL for Talawa mobile app developers to use is:
+The Organization URL for Talawa mobile app developers will depend upon the device on which Mobile app is installed.
 
-       http://localhost:4000/graphql/
+- If Talawa Mobile App is installed on Android Virtual Device (AVD): Use the following URL:
```
Comment (palisadoes) [inline INSTALLATION.md:343]: Add a ### heading for this like:

### On a Virtual Device

File: INSTALLATION.md (hunk 2 above)
Comment (palisadoes) [inline INSTALLATION.md:347]: Add a ### heading for this like:

### On a Real Device

**Review by palisadoes — COMMENTED** [review]
Regenerate the Table of Contents too. Many IDEs have plugins for that.

**Review by palisadoes — CHANGES_REQUESTED** [review]
Regenerate the Table of Contents too. Many IDEs have plugins for that.

File: INSTALLATION.md (hunk 1 above)
Comment (anshgoyalevil) [reply INSTALLATION.md]: Done!

File: INSTALLATION.md (hunk 3 above)
Comment (anshgoyalevil) [reply INSTALLATION.md]: Done!

**Review by palisadoes — APPROVED** [review]

File: INSTALLATION.md (hunk 2 above)
Comment (anshgoyalevil) [reply INSTALLATION.md]: Done!

Comment (anshgoyalevil) [reply INSTALLATION.md]: Done!

**Pull request merged**